processing/src/main/java/org/apache/druid/segment/nested/NestedFieldLiteralDictionaryEncodedColumn.java

```diff
@@ -146,7 +146,7 @@ public String lookupName(int id)
     final int globalId = dictionary.get(id);
     if (globalId < globalDictionary.size()) {
       return StringUtils.fromUtf8Nullable(globalDictionary.get(globalId));
-    } else if (globalId < adjustLongId + globalLongDictionary.size()) {
+    } else if (globalId < globalDictionary.size() + globalLongDictionary.size()) {
```

Comment:
Why are these numbers different from each other?  Or is this change for readability rather than correctness?  If it is for readability, should we not do the same thing in `NestedFieldLiteralColumnIndexSupplier.getIndexes`?

Reply:
i changed for readability, though the other one is consistent in the opposite direction using the adjust values instead, i just wanted the if checks to be using the same thing instead of a mix

Reply:
Does it makes sense to unify it the other way around?  Make this `globalId < adjustDoubleId`?